redfish_guide/topics/using-redfish-physical-infra.adoc

```diff
@@ -195,3 +195,65 @@ the selected image and customization template, has set the servers to boot once
 of Operating System booting and installation, which then takes additional time
 before it finishes.
 
+=== Physical Server Firmware Update
+{product-title} enables performing firmware update of the supported {redfish}
+physical systems. The Firmware Update functionality consists of the following:
+
+* *Firmware Registry integration*. Firmware Registry is an external service that contains
+  a list of binaries that are the payload in the firmware update process. The registry
+  integration provides the inventory of the available firmware binaries tagged by
+  manufacturer and system model. Only compatible binaries are offered to install on a
+  specific server.
+* *Firmware Update request*. Firmware Update process is implemented in a form of
+  {product-title_short} request which can be initialized either from physical server details
+  page or list page. The latter allows one to trigger firmware update process on a number
+  of servers in one go.

```

Comment:
While I agree that reiterating a message from the release notes is useful, I'm not a fan of having copy&pasted text.

Reply:
Yeah. Now that you pointed out, I actually found a neat solution to this: adoc supports "named chunks of text" which you can reuse later with `include::`. I've updated the code to be using that, please see.